rust/lance-index/src/scalar/inverted/index.rs

```diff
@@ -48,32 +74,6 @@ use snafu::location;
 use std::sync::LazyLock;
 use tracing::{info, instrument};
 
-use super::{
-    builder::{
-        doc_file_path, inverted_list_schema, posting_file_path, token_file_path, ScoredDoc,
-        BLOCK_SIZE,
-    },
-    iter::PlainPostingListIterator,
-    query::*,
-    scorer::{idf, BM25Scorer, Scorer, B, K1},
-};
-use super::{
-    builder::{InnerBuilder, PositionRecorder},
-    encoding::compress_posting_list,
-    iter::CompressedPostingListIterator,
-};
```

Comment:
I'm going to assume these import changes are just moving things around (maybe some kind of automated formatting tool?)

Reply:
Yes, they are automated formatted, let me fix them.